index.md

```diff
@@ -0,0 +1,16 @@
+# OpenEBS Jiva Helm Charts
+
+The contents of this branch (gh-pages) are published via GitHub pages at https://openebs.github.io/jiva-operator/
+
+
+<img width="300" align="right" alt="OpenEBS Logo" src="https://raw.githubusercontent.com/cncf/artwork/master/projects/openebs/stacked/color/openebs-stacked-color.png" xmlns="http://www.w3.org/1999/html">
+
+Add OpenEBS Jiva repository to Helm repos:
+
+```bash
+helm repo add openebs-jiva https://openebs.github.io/jiva-operator
+```
+
+## Charts
+
+For installation and upgrading instructions see [Jiva chart readme](https://github.com/openebs/jiva-operator/blob/master/deploy/helm/charts/README.md).
```

Comment:
Can we add configurable flags here?

Reply:
My thought was to not add them here, otherwise will endup maintaining it in 2 places and chances are there that will missed to update in gh-pages.

But i will add the configurable parameters here, similar to other charts we have. And we will find some automation to update the index.md file for any changes in README.